base/exports.jl

```diff
@@ -23,6 +23,7 @@ export
     Docs,
     Markdown,
     Threads,
+    Iterators,
```

Comment:
Ah. But if exported from Base, does `Iterators` refer to something different before you do `using Iterators` vs after?